api-server/package.json

```diff
@@ -5,7 +5,8 @@
   "license": "BSD-3-Clause",
   "private": true,
   "engines": {
-    "node": ">= 14.0.0"
+    "node": ">= 14.0.0",
+    "npm": "^7.7.5"
```

Comment:
We want to target `npm 6.x` explicitly, here and elsewhere. See: https://github.com/freeCodeCamp/freeCodeCamp/pull/41589#issuecomment-808180874

Reply:
Thing is, the reason I've targeted npm 7, is because that tells [renovatebot to use it](https://docs.renovatebot.com/node/#configuring-which-version-of-npm-renovate-uses) so it will generate package-locks with lockfileVersion 2. That's what the renovatebot devs, [recommend too](https://docs.renovatebot.com/dependency-pinning/#what-a-lock-file-doesnt-do-for-you).

> For this reason our usual recommendation using a lock file regardless of whether you pin dependencies or not, and pinning even if you have a lock file.

Reply:
I understand the sentiment. 

However, as part of the ops we would be using the npm version that ships with node. Like I mentioned before npm 7 does not play nice yet because of its gotchas with peer deps. 

We want `npm` to work for us in our Node.js deployments environments and not the other way round. We do not care what users use, we also do not care what the lockfile version is until node ships it by default in lts, its non-standard and is a restriction howsoever tempting the benefits might be.

Let's use npm 6 explicitly. 

Thanks.

Reply:
We should update the DevOps manual, in that case, since it includes `npm i -g npm` which upgrades to 7 now.

I'm fine with sticking to 6 for now, but once we've ironed out any peer dep issues, are you okay with switching everything to 7?  I feel like it has to happen at some point.

Reply:
Thanks again for updating the docs and making the changes. We would be fine with npm 7, when Node.js starts shipping it with LTS. It's a bit opinionated, the point of which is they make that call when there is community wide adoption and its stable enough. 

When they do we should be ready with sorting out the peer-deps (on a branch on this repo) if you so prefer.

Reply:
No problem.  I kind of wanted to get ahead of this, but nothing's actually broken about our current approach and the peer dep issue (I *think* it's only Gatsby that's causing issue) could be gone by the next LTS.  If it ain't broke...